### PR TITLE
Time series - POST and GET

### PR DESCRIPTION
**Pull request by marcel2012**
Example POST requests

```json
{
  "observable_information_id": 6,
  "measure_id": 0,
  "type": "Timestamp",
  "signal_values": [
    {
      "timestamp": 100,
      "value": 10
    },
    {
      "timestamp": 200,
      "value": 20
    }
  ],
  "additional_properties": [
    {
      "key": "string",
      "value": "string"
    }
  ]
}
```

```json
{
  "observable_information_id": 6,
  "measure_id": 0,
  "type": "Epoch",
  "signal_values": [
    {
      "start_timestamp": 10,
      "end_timestamp": 15,
      "value": 10
    },
    {
      "start_timestamp": 25,
      "end_timestamp": 27,
      "value": 20
    }
],
  "additional_properties": [
    {
      "key": "string",
      "value": "string"
    }
  ]
}
````

### CODE REVIEW

File: graph_api/database_service.py [hunk 1]
```diff
@@ -115,6 +117,49 @@ def get_nodes(self, label):
...
+            query (NodeRowsQueryIn): query to search by
+        Returns:
+            Result of request
+        """
+        return_list = []
+        where_list = []
+        match_list = []
+        if query.relations is not None:
+            for relation in query.relations:
+                statement = "(n_" + str(relation.begin_node_index) + ")-["
```
Comment (mrwrob) [inline graph_api/database_service.py:133]: Use f-strings, optionally `format()` to create new strings. 
https://realpython.com/python-f-strings/

File: graph_api/node/node_router.py [hunk 2]
```diff
@@ -60,6 +63,19 @@ async def get_nodes(self, label: str, response: Response):
 
         return nodes
 
+    @router.post("/nodes_query", tags=["nodes"], response_model=NodeRowsOut)
```
Comment (mrwrob) [inline graph_api/node/node_router.py:66]: Why this is `post` not `get`?

File: grisera_api/time_series/time_series_service_graphdb_with_signal_values.py [hunk 3]
```diff
@@ -0,0 +1,341 @@
...
+                    "label": "nextActivityExecution",
+                    "min_count": 0
+                },
+                {
+                    "begin_node_index": 6,
+                    "end_node_index": 5,
+                    "label": "hasScenario"
+                },
+            ]
+        }
```
Comment (mrwrob) [inline grisera_api/time_series/time_series_service_graphdb_with_signal_values.py:93]: Is this an universal query to get time series?

**Review by mrwrob — CHANGES_REQUESTED** [review]

File: graph_api/node/node_router.py (hunk 2 above)
Comment (marcel2012) [reply graph_api/node/node_router.py:66]: I discussed it with @terzawad and POST request with JSON body have benefits:
- clear, human readable JSON format (no GET array parameters)
- query is not limited to 2048 characters

I know that in pure REST API it should be GET.

File: grisera_api/time_series/time_series_service_graphdb_with_signal_values.py (hunk 3 above)
Comment (marcel2012) [reply grisera_api/time_series/time_series_service_graphdb_with_signal_values.py:93]: This is query to get experiment id from time series id. I need this to make common timestamp line connected to the experiment.

File: graph_api/node/node_router.py (hunk 2 above)
Comment (mrwrob) [reply graph_api/node/node_router.py:66]: Got it 👍

File: graph_api/database_service.py (hunk 1 above)
Comment (mrwrob) [reply graph_api/database_service.py]: 👌

**Review by mrwrob — APPROVED** [review]

**Comment by blazejfury** [comment]
Looks fantastic.

**Pull request merged**